File: app/src/main/java/com/qualitasvita/soundmind/Answer.java
package com.qualitasvita.soundmind;

import java.io.Serializable;

/**
 * Класс используется для представления записи эмоции и мысли в удобной форме(текст, значение).
 */
public class Answer implements Serializable {
    private String text;
    private int level;

    public Answer(String text, int level) {
        this.text = text;
        this.level = level;
    }

    public Answer(String text) {
        this.text = text;
        this.level = 0;
    }

    public Answer() {
        this.text = "";
        this.level = 0;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public int getLevel() {
        return level;
    }

    public void setLevel(int level) {
        this.level = level;
    }
}

File: app/src/main/java/com/qualitasvita/soundmind/model/Answer.kt
package com.qualitasvita.soundmind.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.Ignore
import androidx.room.PrimaryKey

/**
 * Класс используется для представления записи эмоции и мысли в удобной форме(текст, значение).
 */
@Entity(tableName = "emotions_negative")
data class Answer(
        @PrimaryKey
        @ColumnInfo(name = "emotion")
        var text: String) {

    @ColumnInfo(name = "emotion_level")
    var level: Int = 0

    @Ignore
    constructor(text: String, level: Int) : this(text) {
        this.text = text
        this.level = level
    }
}